packages/order-watcher/src/utils/assert.ts

```diff
@@ -16,12 +16,14 @@ export const assert = {
         orderHash: string,
         signature: string,
         signerAddress: string,
+        signatureValidationOpts: SignatureValidationOpts,
     ): Promise<void> {
         const isValid = await signatureUtils.isValidSignatureAsync(
             supportedProvider,
             orderHash,
             signature,
             signerAddress,
+            signatureValidationOpts,
```

Comment:
Let's add an entry in the `order-watcher` and `0x.js` CHANGELOGS since both of their public interfaces have changed in a non-breaking way. Minor patches.

Reply:
Updated MINOR version of `0x.js` because of new optional argument in `signatureUtils.isValidSignatureAsync` and PATCH version in `order-watcher` because the public interface did not change (exchange address is taken from the `signedOrder`). Hope you agree?!